src/main/java/net/snowflake/client/core/HttpUtil.java

```diff
@@ -490,6 +534,7 @@ public static RequestConfig getRequestConfigWithoutCookies() {
   }
 
   public static void setRequestConfig(RequestConfig requestConfig) {
+    logger.debug("Setting default request config to: {}", requestConfig.toString());
```

Comment:
please ensure that `requestConfig` is not null

Reply:
Done

Reply:
toString not necessary, I'm sorry for my previous comment